exchanges/execute/src/execute.ts

```diff
@@ -106,6 +101,7 @@ const makeExecuteSource = (
       });
 
     return () => {
+      iterator && iterator.return && iterator.return();
```

Comment:
Just to bring it in line with the codding style of the repo, I'd suggest the following:

```suggestion
      if (iterator && iterator.return) iterator.return();
```

Reply:
Thanks! Done :)